lib/src/Base/Func/EvaluationImplementation.cxx

```diff
@@ -432,14 +432,15 @@ Graph EvaluationImplementation::draw(const UnsignedInteger firstInputMarginal,
   isoValues.buildDefaultLabels();
   const Point levels(isoValues.getLevels());
   const Description labels(isoValues.getLabels());
+  const Description palette = DrawableImplementation::BuildViridisPalette(levels.getDimension());
```

Comment:
why hardcode viridis now that you're adding everything needed to switch colormaps ?

Reply:
The idea was to replace a code using a rainbow colormap to something more appropriate. I admit that replacing a hard-coded palette with another hard-coded palette was not the best option. I see at least to methods to implement your idea:

1) We might use the existing `BuildDefaultPalette` method
```cpp
const Description palette = DrawableImplementation::BuildDefaultPalette(levels.getDimension(), );
```
In this case, the default palette will be Tableau. The user can customize this parameter by setting the 'Drawable-DefaultPaletteName' ResourceMap key. 

2) Assuming that we create a new `BuildPalette(size, paletteName)` where `paletteName` is a `string`, then we could use a default `ResourceMap` key:
```cpp
const Description paletteName = ResourceMap::GetAsString("Evaluation-Palette");
const Description palette = DrawableImplementation::BuildPalette(levels.getDimension(), paletteName);
```
The default value of this key might be Viridis. The user can customize this by setting the "Evaluation-Palette" ResourceMap key.

@jschueller : Which option would fit best in your opinion?